.vscode/launch.json

```diff
@@ -6,7 +6,7 @@
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
-            "args": ["--extensionDevelopmentPath=${workspaceRoot}"],
+            "args": ["--extensionDevelopmentPath=${workspaceRoot}", "--disable-extensions"],
```

Comment:
> "--disable-extensions" [](start = 67, length = 23)

Hmm.. This will prevent me from testing changes to the SARIF extension with our WAVE extensions... Not critical as I can just disable this when I'm doing that, just something to keep in mind. Curious about why this change?

Reply:
It seemed much snappier on startup. It's no big deal for me (I can enable only on my machine) so I've backed it out.

---
In reply to: [452380143](https://github.com/microsoft/sarif-vscode-extension/pull/297#discussion_r452380143) [](ancestors = 452380143)